src/java/org/apache/cassandra/io/util/PathUtils.java

```diff
@@ -334,6 +336,41 @@ private static void deleteRecursive(Path path, RateLimiter rateLimiter, Consumer
         delete(path, rateLimiter);
     }
 
+    /**
+     * Recursively delete the content of the directory, but not the directory itself.
+     */
```

Comment:
Should the comment follow the javadoc pattern as in all other methods as it is part of API?